README.md

```diff
@@ -693,6 +693,17 @@ Specifies the number of seconds an IP address is in cooldown after pod deletion.
 **Note:** 0 is a supported value, however it is highly discouraged.
 **Note:** Higher cooldown periods may lead to a higher number of EC2 API calls as IPs are in cooldown cache.
 
+----
+
+#### `DISABLE_CONTAINER_V6` (v1.14.x+)
```

Comment:
This will not be part of v1.14.0, as we already have the build tag for it. Let's instead make it v1.14.1+